assets/src/auth/pages/RestrictedPage.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from "react";
-import { ErrorAlert } from "@pay/web";
-
-import { usePayUser } from "../../users";
+import { ErrorAlert, Loader } from "@pay/web";
+import { useAuth0 } from "../../auth/AuthContext";
 
 interface Props {
   children: JSX.Element;
 }
 
 const RestrictedPage: React.FC<Props> = ({ children }) => {
-  const user = usePayUser();
+  const { isInitializing, isAuthenticated } = useAuth0();
+
+  if (isInitializing) return <Loader message="Loading " />;
```

Comment:
@acodegirl is your prettier running? this `if` should be surrounded with curly braces?

Reply:
@acodegirl you have a trailing space in the `message`